nat.go

```diff
@@ -60,32 +59,31 @@ func (odbi *ovndb) rowToNat(uuid string) *NAT {
 			nat.LogicalIP = lip.(libovsdb.UUID).GoUUID
 		case string:
 			nat.LogicalIP = lip.(string)
-		default:
 		}
 
 	}
 
 	return nat
 }
 
-func (odbi *ovndb) lrNatAddImp(lr string, ntype string, externalIp string, externalMac string, logicalIp string, logicalPort string, external_ids map[string]string) (*OvnCommand, error) {
+func (odbi *ovndb) lrNatAddImp(ROUTER string, TYPE string, EXTERNAL_IP string, LOGICAL_IP string, external_ids map[string]string, LogicalPortAndExternalMac ...string) (*OvnCommand, error) {
```

Comment:
please, not use UPPERCASE, use lowercase.

Reply:
sorry sorry , I misunderstand what you mean.

Reply:
all function args always started lowercase, 
so not TYPE but, ntype or natType or something like this. Also in go _ not used, so localIP externalIP

Reply:
thanks , I don't have experience for open source project and code format before , you really teach me so much things,thanks again.